src/demo/TinhToan.java

```diff
@@ -0,0 +1,314 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package demo;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ *
+ * @author dell
+ */
+public class TinhToan extends javax.swing.JFrame {
+    double a,b;
+    /** 
+     * Creates new form TinhToan
+     */
+    public TinhToan() {
+        initComponents();
+        setLocationRelativeTo(null);
+        
+        RaBtn_cong.setSelected(true);
+        if(Radientichhcn.isSelected()||Radientichhcn.isSelected()){
+                jLabel2.setText("Chiều rộng");
+                jLabel3.setText("Chiều dài");
+        }
+    }
+//        Hàm kiểm tra rỗng
+        public boolean kiemtrarong(){
+            if(txtA.getText()=="" || txtB.getText()==""){
+                return false;
+            }
+            return true;
+        }
+
+
+    /**
+     * This method is called from within the constructor to initialize the form.
+     * WARNING: Do NOT modify this code. The content of this method is always
+     * regenerated by the Form Editor.
+     */
+    @SuppressWarnings("unchecked")
+    // <editor-fold defaultstate="collapsed" desc="Generated Code">//GEN-BEGIN:initComponents
+    private void initComponents() {
+
+        jRadioButtonMenuItem1 = new javax.swing.JRadioButtonMenuItem();
+        btnGR = new javax.swing.ButtonGroup();
+        jLabel1 = new javax.swing.JLabel();
+        jLabel2 = new javax.swing.JLabel();
+        jLabel3 = new javax.swing.JLabel();
+        txtA = new javax.swing.JTextField();
+        txtB = new javax.swing.JTextField();
+        jLabel4 = new javax.swing.JLabel();
+        txtKQ = new javax.swing.JTextField();
+        RaBtn_tru = new javax.swing.JRadioButton();
+        RaBtn_cong = new javax.swing.JRadioButton();
+        RaBtn_nhan = new javax.swing.JRadioButton();
+        RaBtn_chia = new javax.swing.JRadioButton();
+        Rachuvi = new javax.swing.JRadioButton();
+        Radientichhcn = new javax.swing.JRadioButton();
+        Radientichhv = new javax.swing.JRadioButton();
```

Comment:
tên biến chưa chuẩn, cần đổi sang dienTichHCN và còn lại

Reply:
ok, accept it